### PR TITLE
Fix broken URL for "Deep Dive TiKV"

### PR DESCRIPTION
**Pull request by Himself65**
###  What have you changed?

Please explain in detail what the changes are in this PR and why they are needed:

- Summarize your change
- How does this PR work? Include a brief introduction for any changed logic (optional)
- Separately describe each logical change (optional)
- Describe any limitations of the current code (optional)

Don't assume reviewers understand the original issue.

###  What is the type of the changes?

- New feature (a change which adds functionality)
- Improvement (a change which is an improvement to an existing feature)
- Bugfix (a change which fixes an issue)
- Engineering (engineering change which doesn't change any feature or fix any issue)
- Misc (other changes)

###  How is the PR tested?

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ See [Governance](https://github.com/tikv/community/blob/master/GOVERNANCE.md).
 
 ## Documentation
 
-For instructions on deployment, configuration, and maintenance of TiKV,see TiKV documentation on our [website](https://tikv.org/docs/3.0/tasks/introduction/). For more details on concepts and designs behind TiKV, see [Deep Dive TiKV](https://tikv.org/docs/deep-dive/introduction/).
+For instructions on deployment, configuration, and maintenance of TiKV,see TiKV documentation on our [website](https://tikv.org/docs/3.0/tasks/introduction/). For more details on concepts and designs behind TiKV, see [Deep Dive TiKV](https://tikv.org/deep-dive/introduction/).
```
Comment (siddontang) [inline README.md]: PTAL @dcalvin

Comment (dcalvin) [reply README.md:51]: This change is not required as we have an alias at work now. Both links will work.

**Comment by dcalvin** [comment]
We can close this PR as this change is not required for now.

**Pull request closed without merging**